tests/exit_methods.pyi

```diff
@@ -21,7 +21,7 @@ class GoodTwo:
     async def __aexit__(self, typ: Type[BaseException] | None, *args: object) -> bool: ...
 
 class GoodThree:
-    def __exit__(self, __typ: typing.Type[BaseException] | None, exc: BaseException | None, *args: object) -> None: ...
+    def __exit__(self, __typ: typing.Type[BaseException] | None, exc: BaseException | None, *args: object) -> None: ...  # Y063 Use PEP-570 syntax to indicate positional-only arguments
```

Comment:
Considering Y063 is not the focus of this test, this could be converted to PEP 570 syntax as well.

Reply:
I considered this, but I actually like having examples in our test suite that trigger multiple rules, so that we can see how the rules interact with each other. There have been times in the past where that's led me to realise that different rules were giving contradictory recommendations on the same line of code, which has then caused me to rethink

Reply:
In that case, having a separate test case that makes this explicit would be better, in my opinion.

Reply:
Fair enough, I'll make that change!